src/openassetio-core/include/openassetio/managerApi/ManagerInterface.hpp

```diff
@@ -888,6 +888,8 @@ class OPENASSETIO_CORE_EXPORT ManagerInterface {
    *
    * @param context The calling context.
    *
+   * @param hostSession The API session.
+   *
```

Comment:
[fix] Can we use the standard desc for this as it highlights its use for logging?

Reply:
There are lots of other methods that simply say "The API session".  But I see the text you mean, will switch.

Reply:
Used the more descriptive form in 3e1807f0